test/extended/operators/certs.go

```diff
@@ -79,6 +78,7 @@ func gatherCertsFromPlatformNamespaces(ctx context.Context, kubeClient kubernete
 	return certgraphanalysis.GatherCertsFromPlatformNamespaces(ctx, kubeClient,
 		certgraphanalysis.SkipRevisioned,
 		certgraphanalysis.SkipHashed,
+		certgraphanalysis.SkipBootstrapCerts(bootstrapIP),
```

Comment:
at some point we'll need to unskip these.

Reply:
Created https://issues.redhat.com/browse/API-1801 to follow up